Dockerfile

```diff
@@ -27,9 +30,12 @@ RUN python3 -m pip install python-lsp-server
 
 COPY --from=jaeger-builder /go/bin/all-in-one-linux /usr/bin/
 
+COPY --from=yq-builder /usr/bin/yq /usr/bin/yq
```

Comment:
Thinking out loud here, but I think there was talk of doing some dependency management for external providers a while back. It's fine for now, but this method could bloat the Dockerfile when adding more providers.

Reply:
It will for sure, I think we should ask @dymurray to prioritize this work for the next version.